packages/Builtins/_.candy

```diff
@@ -77,164 +106,382 @@ fitsInRustU128 value =
   value | isLessThanOrEqualTo rustU128Max
 
 intAdd a b :=
+  # Returns `a` + `b`.
+  #
+  # ```
+  # intAdd 1 2 => 3
+  # ```
   needs (a | typeIs Int)
   needs (b | typeIs Int)
   ✨.intAdd a b
+
 intBitLength value :=
-  # Determines the fewest bits necessary to express this integer, not including the sign.
+  # Returns the number of bits that are necessary to represent that integer,
+  # ignoring the sign.
+  #
+  # ```
+  # intBitLength 1 => 1
+  # intBitLength 2 => 2
+  # intBitLength 3 => 2
+  # intBitLength -3 => 2
+  # ```
   needs (value | typeIs Int)
   ✨.intBitLength value
+
 intBitwiseAnd a b :=
+  # Returns the result of taking the bitwise "and" of `a` and `b`.
+  #
+  # ```
+  # intBitwiseAnd 1 2 => 1
+  # ```
   needs (a | typeIs Int)
   needs (b | typeIs Int)
   ✨.intBitwiseAnd a b
+
 intBitwiseOr a b :=
+  # Returns the result of taking the bitwise "or" of `a` and `b`.
+  #
+  # ```
+  # intBitwiseOr 1 2 => 2
+  # ```
   needs (a | typeIs Int)
   needs (b | typeIs Int)
   ✨.intBitwiseOr a b
+
 intBitwiseXor a b :=
+  # Returns the result of taking the bitwise "xor" of `a` and `b`.
+  #
+  # ```
+  # intBitwiseXor 1 3 => 2
+  # ```
   needs (a | typeIs Int)
   needs (b | typeIs Int)
   ✨.intBitwiseXor a b
+
 intCompareTo a b :=
-  # Returns `Less`, `Equal`, or `Greater` depending on whether `a` is less than, equal to, or
-  # greater than `b`.
+  # Returns the relationship between the integers as a tag, which is either
+  # `Less`, `Equal`, or `Greater`.
+  #
+  # ```
+  # intCompareTo 1 3 => Less
+  # intCompareTo 3 3 => Equal
+  # intCompareTo 5 3 => Greater
+  # ```
   needs (a | typeIs Int)
   needs (b | typeIs Int)
   ✨.intCompareTo a b
+
 intDivideTruncating dividend divisor :=
   # Returns `dividend` ÷ `divisor`.
   #
-  # This operation rounds towards zero, truncating any fractional part of the exact result.
+  # This operation rounds towards zero, truncating any fractional part of the
```

Comment:
Maybe mention that this operation is for division. For all other operations, this is mentioned first and might make it sound like this builtin is for rounding.